aten/src/ATen/native/cuda/Normalization.cuh

```diff
@@ -1512,4 +1625,62 @@ at::Tensor batch_norm_backward_elemt_channels_last_cuda_template(
   return grad_input;
 }
 
+at::Tensor batch_norm_backward_elemt_channels_last_cuda_template(
```

Comment:
this is, mostly, same function as above, but without `count` argument?

Reply:
Yes, `count` is only used by `SyncBatchNorm`.